### PR TITLE
Issue/159/iterated read

### PR DESCRIPTION
**Pull request by eacharles**

### CODE REVIEW

**Review by drewoldag — APPROVED** [review]
From what I can tell, this is a utility that will return the number of pdfs in a hdf5 file, yeah? If so, it looks good to me.

File: src/qp/factory.py [hunk 1]
```diff
@@ -164,6 +164,20 @@ def read(self, filename):
...
+
+        Parameters
+        ----------
+        filename : `str`
+
+        Returns
+        -------
+        nrows : `int`
+        """
+        f, infp = io.readHdf5Group(filename, 'data')
```
Comment (drewoldag) [inline src/qp/factory.py:178]: Is `infp` going to be used for something at a later date?

Comment (eacharles) [reply src/qp/factory.py:178]: Not it this function.   I'll set it to `f, _` instead.

**Review by sschmidt23 — APPROVED** [review]
Looks good to me

**Pull request merged**